### DISCUSSION TITLE
Fork for new idea

### DISCUSSION THREAD

**Discussion by secondl1ght** [dennisreimann/bitcoin-donations#115] [Ideas]
Hi there,

I have an idea to fork this site and create a complimentary site where developers can go to find open source bitcoin projects that are looking for contributors. If there is already a site like this then please let me know, but I thought it would be cool and a good resource to have a website similar to this one except the cards could be projects for devs to work on.

Please let me know if you are OK with me forking for this purpose. I came up with the idea because I personally am a junior developer and I am looking for open source bitcoin projects that I can take part with. Having many good ones in one place would be a helpful resource I think. 

The only part I am not sure about is how to get the word out to different projects and see if they would like to be added to the list. I could just add a bunch of projects that I am aware of, but I thought it might be good to add projects that are actively looking for contributors. 

Anyway let me know what you think, thanks!

**Comment by dennisreimann** [comment]
Sure, go ahead! All the best :+1:

**Reply by secondl1ght** [reply]
Awesome thanks!
